### ISSUE TITLE
Can't connect to ejabberd with TLS

### ISSUE THREAD

**Issue by pztrn** [movim/movim#236]
I have `starttls_required: true` in ejabberd configuration, and denying protocols up to tlsv1.1, and movim failed to connect with:

```
2016-08-27 00:14:49.578 [debug] <0.1627.0>@ejabberd_receiver:handle_info:194 TLS error = SSL_do_handshake failed: error:140760FC:SSL routines:SSL23_GET_CLIENT_HELLO:unknown protocol
```

When I disable it - I can connect, but I'm loosing ability to login with other clients (still IDK why). But point is: Movim cannot connect to XMPP server with forced TLS, and still trying to use SSLv2 or SSLv3.

**Comment by edhelas** [comment]
I'll try to have a look. Do you have credentials for a test account on your XMPP to help me to reproduce the issue on my side?

**Comment by pztrn** [comment]
I have no test server for now, but you can ping me in muc when you'll be ready to test and debug, and I will force tls and create account for you :)

**Comment by pztrn** [comment]
Fixed in fd732155a5e6d266675dd0a7da36d854098f0e63
